packages/@remirror/extension-annotation/package.json

```
@@ -28,10 +28,12 @@
   },
   "devDependencies": {
     "@remirror/core": "1.0.0-next.22",
+    "@remirror/extension-positioner": "*",
```

I think the linting is failing due to this. I'm not sure why `pnpm fix` sometimes doesn't fix things properly.

```suggestion
    "@remirror/extension-positioner": "1.0.0-next.22",
```

---

My bad, I forgot that it wouldn't automatically update the lock file. This is a problem with auto suggestions. They seldom work without further adjustments 😄